build/common/installer/scripts/tomlparser.rb

```diff
@@ -205,6 +206,17 @@ def populateSettingValuesFromConfigMap(parsedConfig)
     rescue => errorStr
       ConfigParseErrorLogger.logError("Exception while reading config map settings for adx database name - #{errorStr}, using default #{@adxDatabaseName}, please check config map for errors")
     end
+
+    #Get Kubernetes Metadata setting
+    begin
+      if !parsedConfig[:log_collection_settings][:metadata_collection].nil? && !parsedConfig[:log_collection_settings][:metadata_collection][:enabled].nil?
+        @logEnableKubernetesMetadata = parsedConfig[:log_collection_settings][:metadata_collection][:enabled]
+        puts "config::Using config map setting for kubernetes metadata"
+
+        if @containerLogSchemaVersion.strip.casecmp("v2") != 0
```

Comment:
if customer did not set containerlog schema but enabled it through DCR, we still need to support this feature

Reply:
i can remove this check from here assuming we are doing the check later in oms.go